.eslintrc.json

```diff
@@ -111,6 +112,7 @@
       }
     ],
     "no-use-before-define": 0,
+    "no-restricted-syntax": ["error", "ForInStatement", "LabeledStatement", "WithStatement"],
```

Comment:
this project is for the server-side, we can use `for (const a of b) {…}` because it’s 2023